pages/linux/openvpn3.md

```diff
@@ -0,0 +1,24 @@
+# openvpn3
+
+> OpenVPN 3 Linux client.
+> More information: <https://community.openvpn.net/openvpn/wiki/OpenVPN3Linux>.
+
+- Start new VPN session:
+
+`openvpn3 session-start --config my-vpn-config.conf`
```

Comment:
```suggestion
`openvpn3 session-start --config {{path/to/config.conf}}`
```
This is how we indicate that you have to specify the path and not just the name. Also, the `{{ }}` enable syntax highlighting.

Reply:
great! didnt realize this had diff syntax highlight. updated all examples